### PR TITLE
Alert time range

### PR DESCRIPTION
**Pull request by 121watts**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass

Connect #1030 

### The problem
We get all the alerts since the beginning of time.  This causes the browser to hangup and crash and is generally a 💩  experience.

### The Solution
Having a default timeRange and the ability for a user to update the timeRange gets us 80% of the way there.

### CODE REVIEW

File: ui/src/alerts/containers/AlertsApp.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import NoKapacitorError from '../../shared/components/NoKapacitorError'
+import CustomTimeRange from '../../shared/components/CustomTimeRange'
+
+import {getAlerts} from '../apis'
+import AJAX from 'utils/ajax'
+
+import _ from 'lodash'
+import moment from 'moment'
 
 const AlertsApp = React.createClass({
```
Comment (cryptoquick) [inline ui/src/alerts/containers/AlertsApp.js]: Do you think this component should be updated?

**Comment by cryptoquick** [comment]
@goller yeah, I'd prefer to use yarn's license command. It should probably go like this:
`yarn licenses generate-disclaimer > LICENSE_OF_DEPENDENCIES.md`

File: ui/src/alerts/containers/AlertsApp.js (hunk 1 above)
Comment (121watts) [inline ui/src/alerts/containers/AlertsApp.js]: Totally.  I'll take a few minutes and update it to a js class.

**Comment by cryptoquick** [comment]
Maybe we can switch to yarn's version in a different branch, since the yarn command will generate it in a different format.

**Comment by alexpaxton** [comment]
custom-time-range

It has been styled!

**Pull request merged**